static/styles/settings.scss

```diff
@@ -1600,10 +1600,6 @@ input[type=checkbox] {
                 border-bottom: none;
             }
 
-            &.no-border {
-                border-color: transparent;
-            }
-
```

Comment:
What's the story with this change?  If the class was useless, we should remove the code that adds it (and do it in its own commit separate from these other fixes).

Reply:
The difference `.no-border` made was only one pixel, which was barely noticable since both the border and the background have the same color. In single column mode we no longer display the background so `.no-border` led to a missing border ... which looked broken. I decided to remove it (and now also removed the code that adds it) since I found overriding this class in the media query to not be worth it since effectively nobody notices it anyway.